SigningService/Agents/SignAgent.cs

```diff
@@ -45,7 +45,7 @@ private void SignPackagePart(PackagePart packagePart)
         {
             foreach (var signer in _signerRepository)
             {
-                signer.TrySign(packagePart);
+                throw new NotImplementedException();
```

Comment:
I removed this as I currently pass stream in the constructor. Need to revert it when I have time.